docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -45,6 +45,7 @@ With this release, we’re streamlining the installation process for OpenNMS on
 
 Before upgrading, we strongly recommend backing up your existing installation including configuration files, databases, and any custom scripts or integrations to prevent data loss or disruption.
 
+To preserve local changes in `/etc/opennms`, copy that folder to `/opt/opennms/etc` before upgrading, and preserve file ownership and permissions.
```

Comment:
What about adding this here:

You can prepare the upgrade with the following commands before upgrade:

```
sudo install -d -g opennms -o opennms /opt/opennms/etc
sudo -u opennms cp -r /etc/opennms/* /opt/opennms/etc
```